src/test/java/io/reactivex/rxjava3/observers/SerializedObserverTest.java

```diff
@@ -340,11 +340,11 @@ public void onNext(String t) {
      *
      * When using SynchronizedSubscriber we get this output:
      *
-     * p1: 18 p2: 68 => should be close to each other unless we have thread starvation
+     * {@code p1: 18 p2: 68 => should be close to each other unless we have thread starvation}
```

Comment:
No reason to include "should be close to each other unless we have thread starvation" in the code block.

Reply:
You're correct, just did the whole line for cosmetic reasons, but I can surely remove the rest of the statement from it.